modules/ROOT/pages/general-configuration-guide/attribution-requirements.adoc

```diff
@@ -25,7 +25,7 @@ The {productname} status bar provides the required attribution on the right-hand
 
 The following example shows the status bar product attribution in the bottom, right-hand corner:
 
-image:tinymce5-homepage-demo.png[Screenshot of homepage {productname} demo including product attribution in the bottom, right-hand corner]
+image::tinymce5-homepage-demo.png[Screenshot of homepage {productname} demo including product attribution in the bottom, right-hand corner]
```

Comment:
You've changed this when it was originally correct... this is an inline image so it should be using `image:`.

- [ ] Revert the recent change made to this